src/fragments/lib/push-notifications/android/register_device/10_register_with_pinpoint.mdx

```diff
@@ -0,0 +1,97 @@
+<Callout>
+
+You do not need to add these APIs to your app if you want to use Amplify with Amazon Pinpoint. You should use these APIs if you are interested in supporting multiple push providers.
```

Comment:
Same goes to here as before. Do we mean all these are already provided to us? If we don't need it, why isn't this under advanced workflows? @abdallahshaban557 what do you say?